spec/cirro_io/client/payout_spec.rb

```diff
@@ -15,8 +15,8 @@
     let(:payout) { described_class.create app_worker: app_worker, **attributes }
     let(:attributes) do
       {
-        title: Faker::Hipster.sentence,
-        description: Faker::Hipster.paragraph,
+        title: Faker::Lorem.sentence,
```

Comment:
Changed to `Lorem` cause of escaping issue, that is led to flaky tests
F.E. 
`pbr\\u0026b != pbr&b`